### DISCUSSION TITLE
how should we annotate occluded object which is splitted in to two or more parts?

### DISCUSSION THREAD

**Discussion by xoofee** [ultralytics/yolov5#10053] [General]
This image is from coco
https://filebox.ece.vt.edu/~aroma/web/coco_images/val2014/COCO_val2014_000000005193.jpg
![image](https://user-images.githubusercontent.com/7467604/200175707-4bd06cb7-51fc-4ebf-b0da-726cd787a011.png)

the original annotation of person is:
![image](https://user-images.githubusercontent.com/7467604/200175785-2eb22d04-fdad-4188-8344-50721138a9b6.png)

you can see that PERSON 1 annotation box does not contains his left palm, while PERSON 2 annotation box contains the left palm but not the right palm

does this lead to learning difficulty?

in fact the Yolov5x model's result on this image has a False Positive of Person which has a low IOU with the ground truth. The prediction does not contain PERSON 2's left hand. However, it contains the left hand of PERSON 1, which is wrong
![image](https://user-images.githubusercontent.com/7467604/200175962-e06d4690-3728-4ee1-9768-70ebf74f9d89.png)
`python detect.py --weight yolov5x.pt --source coco-2017/data/000000005193.jpg`

**Comment by glenn-jocher** [comment]
@xoofee annotations are correct, but of course this will be very difficult to predict correctly as these types of annotations represent a minority of the dataset.

**Reply by xoofee** [reply]
I got it! thanks!
